### PR TITLE
Add viewforum_body_topicrow_before template event

### PR DESCRIPTION
**Pull request by Senky**
[PHPBB3-14469](https://tracker.phpbb.com/browse/PHPBB3-14469)

### CODE REVIEW

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -2227,6 +2227,14 @@ viewforum_body_topic_row_prepend
 * Since: 3.1.7-RC1
 * Purpose: Add content at the end of the topic list item.
 
+viewforum_body_topicrow_before
```
Comment (marc1706) [inline phpBB/docs/events.md]: To be consistent with the other events regarding the topicrow, we should probably change it to `viewforum_body_topic`_row_before` and move it accordingly before the entries above this one to have a proper alphabetical order.

**Comment by Elsensee** [comment]
events.md is in alphabetical order. Thus `viewforum_body_topicrow_before` needs to be moved up, so it's before `viewforum_body_topicrow_prepend`.

File: phpBB/docs/events.md [hunk 2]
```diff
@@ -2227,6 +2227,14 @@ viewforum_body_topic_row_prepend
 * Since: 3.1.7-RC1
 * Purpose: Add content at the end of the topic list item.
 
+viewforum_body_topicrow_row_before
```
Comment (3D-I) [inline phpBB/docs/events.md]: Shouldn't it be **viewforum_body_topicrow_before** ¿?

Applies to all of this files.

**Comment by 3D-I** [comment]
I added a comment to the commit.

It should be **viewforum_body_topicrow_before** and not **viewforum_body_topicrow_row_before**, I guess.

**Pull request merged**